### DISCUSSION TITLE
How can I change the author profile icon?

### DISCUSSION THREAD

**Discussion by DIAMANUEL** [binokochumolvarghese/lightbi-hugo#66] [General]
Hi, I want to change the icon of the author, but when I search the image it shows an HTML file inside the themes/lightbi-hugo folder, so if i change something there, I'll have to reinstall the theme.
I found this line in the hugo.toml file, but I don't know what is "bottom"

previewCardImagePlacement = "bottom"

**Comment by binokochumolvarghese** [comment]
Hi,
First you have to pull the latest version of the theme. In the same I updated Author parameters as below:

```
[Params.Author]
  name = "Bino Kochumol Varghese"
  website = "https://www.binovarghese.com/"
  authorInfo = "Author short bio"
  authorPostImage = "/img/author-post-img.jpg"
```

You can update the image in the asset folder (assets/img/author-post-img) or update the parameter with required image. For more check the live demo [here](https://github.com/binokochumolvarghese/lightbi-hugo-demo).
